src/components/executive-education-2u/styles/_ExecutiveEducation2UErrorPage.scss

```diff
@@ -0,0 +1,61 @@
+$heading-red: #D23228;
```

Comment:
Use the existing SCSS variables provided by the [Paragon design system](https://paragon-openedx.netlify.app/foundations/colors):

* `$heading-red` -> `$brand-500`
* `$heading-black` -> `$dark-700`
*  `$error-chip-background-color` -> `$white`
* `$error-chip-border-color` -> `$light-300`

Generally a code smell to hardcode custom HEX values as it makes theming difficult.

Reply:
I have used the suggested SCSS variables.